tests/containers/bci_version_check.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Checks if the container version for the test run is still up-to-date
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use utils qw(zypper_call script_retry);
+use db_utils qw(push_image_data_to_db);
+use containers::common;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    select_serial_terminal;
+
+    return unless (get_var('CONTAINER_IMAGE_TO_TEST') && get_var('CONTAINER_IMAGE_BUILD'));
```

Comment:
This should rather end up in the `lib/main_common.pm`

Reply:
To me seems that, if the `unless` check was intended to end the test, it should run before the termnal setting: isn't it?

Reply:
Moreover the new `unless` logic is different from the original 'if` check where the only CONTAINER_IMAGE_TO_TEST missing caused the full code block to skip and the CONTAINER_IMAGE_BUILD missing only skipped an inner sub-block.

Reply:
You both are right in your own way, but that was not the intention of this check.

Leaving this here allows to clone the test run and modify/delete the corresponding values without having the cloned test fail. This is especially useful when disabling `CONTAINER_IMAGE_BUILD`, and likely also for `CONTAINER_IMAGE_TO_TEST`.

I still like @mloviska suggestion and will add it to `lib/main_containers` additionally. But I prefer to have this check here as well, because in the past I encountered some corner cases, where the scheduling-based exclusion was not working as expected.